src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/application.ts

```diff
@@ -138,10 +139,13 @@ export const useApplicationStore = defineStore("application", {
         case "FiveYearWithIteRegistrant":
         case "FiveYearWithSneRegistrant":
         case "FiveYearWithIteAndSneRegistrant":
+          return applicationWizardFiveYear;
         case "IteRegistrant":
+          return applicationWizardIteSne;
         case "SneRegistrant":
+          return applicationWizardIteSne;
         case "IteAndSneRegistrant":
-          return applicationWizardFiveYear;
+          return applicationWizardIteSne;
```

Comment:
tiny nitpik 

we could combine this so we know they all result in the same config

case "IteRegistrant":
case "SneRegistrant":
case "IteAndSneRegistrant":
      return applicationWizardIteSne;